rbac/management/group/view.py

```diff
@@ -59,12 +61,37 @@ def username_filter(queryset, field, value):
         filtered_set = queryset.filter(**filters)
         return filtered_set | Group.platform_default_set()
 
+    def roles_filter(self, queryset, field, value):
+        """Filter for group to lookup list of role name."""
+        if not value:
+            key = 'groups_filter'
+            message = 'No value of roles provided to filter groups!'
+            error = {
+                key: [_(message)]
+            }
+            raise serializers.ValidationError(error)
+        roles_list = value.split(',')
+
+        discriminator = validate_and_get_key(
+            self.request.query_params,
+            ROLE_DISCRIMINATOR_KEY,
+            VALID_ROLE_ROLE_DISCRIMINATOR,
+            'any')
+
+        if discriminator == 'any':
+            return queryset.filter(policies__roles__name__in=roles_list)
```

Comment:
Just curious, is this expected to be a partial match? Other filters work that way, but the expectation of this may be different?

Reply:
Good point. From our conversation, it sounds like they want an exact match, but let me confirm with their use case.

Reply:
Sounds good! I think the exception is `/principals/?usernames=` which is an exact match on a list.

Reply:
Confirmed with them and they want an exact match but case insensitive.

I have updated the code and test to be case insensitive